core/storage/statistics/gae_models.py

```diff
@@ -973,9 +973,9 @@ class PlaythroughModel(base_models.BaseModel):
     exp_id = ndb.StringProperty(indexed=True)
     # Version of the exploration.
     exp_version = ndb.IntegerProperty(indexed=True)
-    # ID of the issue.
-    issue_id = ndb.StringProperty(indexed=True)
-    # The customization args dict for the given issue_id.
+    # Type of the issue.
+    issue_type = ndb.StringProperty(indexed=True)
+    # The customization args dict for the given issue_type.
     issue_customization_args = ndb.JsonProperty(default=None)
     # The playthrough actions for this playthrough. This will be a list of dicts
```

Comment:
Should be ndb.JsonProperty(repeated=True) to signify list

Reply:
Done.